### ISSUE TITLE
Can't claim bonuses through a mobile device

### ISSUE THREAD

**Issue by diwash007** [novuhq/hacksquad-website#54]
![Screenshot_2022-10-06-04-46-27-101_com android chrome](https://user-images.githubusercontent.com/39879720/194179385-30407491-2680-4303-a8f8-9348a1a23817.jpg)

**Comment by nevo-david** [comment]
Do you know how to fix it?

**Comment by diwash007** [comment]
Yeah, i ll dive deep into tailwindcss and try to find a better solution
You may assign it to me.
